Starter-Code/app.js

```diff
@@ -1,41 +1,53 @@
-let choreList = [];
+let choreList = {};
 
 $(document).ready(function(){
   $("#cat").fadeIn(4000);
+  
  
-	// const chores = function (event) {
-	$("#submitButton").click((event) => {
-	event.preventDefault();	
-	if($('#form1').val()){
-		let getChore = $('#form1').val();
-		let newChore = $('<li>');
-		// newChore.innerHTML = getChore;
-		$("#list").append(newChore);
-		$(newChore).html("Cheshire's chore is " + getChore);
-		$('#form1').val('');
-}
-	
-})
+
 
 });
 
 
 
 const addToLocalStorage = (item) => {
-	choreList.push(item)
-	localStorage.setItem("LSChores", JSON.stringify(choreList));
-	console.log("adding to storage");
-};
+	localStorage.setItem('choreList', JSON.stringify(item));
+}
 
 const getFromLocalStorage = () => {
-	console.log("getting from local storage");
-	choreList = JSON.parse(localStorage.getItem("LSChores")) || [];
+	let savedChore = localStorage.getItem('choreList')
+	if (savedChore){
+		choreList = JSON.parse(savedChore);
```

Comment:
You'll want to do

```
choreList = JSON.parse(savedChore) || []
```

This is to catch the situation where it is null/not existing in localStorage yet.